backend/homework/1/app/src/main/java/Main.java

```diff
@@ -0,0 +1,120 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.*;
+
+import java.util.*;
+
+public class Main {
+
+//    private static final Logger logger
```

Comment:
Never leave commented code unnecessarily. Or add a separate comment to describe why it is left on purpose.